### PR TITLE
Fix #12 (interface to K8s script or OpenShift service)

### LINKED ISSUE #12: Add a code which can act as an interface to K8s script or OpenShift service
Having a single interface to handle setup Kubernetes, OpenShift etc helps to improve the user experience. The interface can be called from Vagrantfile or inside the Vagrantbox

### PR DESCRIPTION
**Pull request by praveenkumar**
Initial work of this script taken from https://github.com/LalatenduMohanty/k8s-setup/blob/master/k8s-setup.sh and after adding openshift bits we can now use it as a switch script for k8s and openshift.

### CODE REVIEW

File: utils/sccli.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# sccli - Service Change CLI
+# Script to switch openshift from k8s or vice-versa 
+
+function single_node_setup ()
```
Comment (LalatenduMohanty) [inline utils/sccli.sh]: This function name should be changed to single_node_kube_setup() as this function only handles k8s

**Comment by LalatenduMohanty** [comment]
@praveenkumar this PR should accompany change for RPM specfile. we should copy the script to /opt/adb and create a symlinlk  in /usr/local/bin

File: adb-utils.spec [hunk 2]
```diff
@@ -31,14 +32,19 @@ specific service and directly including it to kickstart file.
 %{buildroot}%{_sysconfdir}/sysconfig/openshift_option
 %{__cp} services/openshift/scripts/* %{buildroot}/opt/adb/openshift/
 %{__cp} utils/* %{buildroot}/opt/adb/
+ln -s /opt/adb/sccli.sh %{buildroot}%{_bindir}/sccli
 
 %files
 %{_sysconfdir}/sysconfig/openshift_option
 %{_unitdir}/openshift.service
+%{_bindir}/sccli
```
Comment (LalatenduMohanty) [inline adb-utils.spec:40]: It would resolve in to /usr/bin/sccli .  What is a better location /usr/bin or /usr/local/bin for sccli?

Comment (LalatenduMohanty) [inline adb-utils.spec:40]: I think we should add sccli to /usr/local/bin as /usr/bin is distribution managed and /usr/local/bin is user managed

Comment (praveenkumar) [inline adb-utils.spec:40]: It is symlink so `/usr/bin` is good enough as per my understanding.

**Pull request merged**